localization/english/religion/wc_religion_draconism_l_english.yml

```diff
@@ -0,0 +1,207 @@
+ neltharism_health_god_name:0 "$draconic_good_god_khazgoroth$"
+ neltharism_health_god_name_possessive:0 "$draconic_good_god_khazgoroth_possessive$"
+ neltharism_fertility_god_name:0 "$draconic_good_god_khazgoroth$"
+ neltharism_fertility_god_name_possessive:0 "$draconic_good_god_khazgoroth_possessive$"
+
+ malygosism:0 "Malygosism"
+ malygosism_adj:0 "Malygosic"
+ malygosism_adherent:0 "Malygosist"
+ malygosism_adherent_plural:0 "Malygosists"
+ malygosism_desc:0 "The followers of the Blue Dragonflight are the majestic guardians of the arcane and the keepers of magic. They believe that the wise Aspect of Magic entrusted the blue dragons with the duty of protecting the magical energies of the world and promoting responsible use of magic. Their faith emphasizes understanding and wielding the power of the arcane for the greater good. The blue dragons draw their power from the Nexus, where the leylines converge. For the followers of the Blue Dragonflight, magic is a potent force that must be respected and used with care. They work to hold those who wield magic accountable for their actions and ensure its responsible use."
+ malygosism_high_god_name:0 "$draconic_good_god_norgannon$"
+ malygosism_high_god_name_2:0 "$malygosism_high_god_name$" # Optional translation alt
+ malygosism_high_god_name_3:0 "$malygosism_high_god_name$" # Optional translation alt
+ malygosism_high_god_name_possessive:0 "$draconic_good_god_norgannon_possessive$"
+ malygosism_high_god_alternate:0 "the Dreamweaver"
+ malygosism_high_god_alternate_possessive:0 "the Dreamweaver's"
+ malygosism_creator_god_name:0 "$draconic_good_god_norgannon$"
+ malygosism_creator_god_name_possessive:0 "$draconic_good_god_norgannon_possessive$"
+ malygosism_health_god_name:0 "$draconic_good_god_norgannon$"
+ malygosism_health_god_name_possessive:0 "$draconic_good_god_norgannon_possessive$"
+ malygosism_fertility_god_name:0 "$draconic_good_god_norgannon$"
+ malygosism_fertility_god_name_possessive:0 "$draconic_good_god_norgannon_possessive$"
+ malygosism_household_god_name:0 "$draconic_good_god_norgannon$"
+ malygosism_household_god_name_possessive:0 "$draconic_good_god_norgannon_possessive$"
+
+ nozdormism:0 "Nozdormism"
+ nozdormism_adj:0 "Nozdormic"
+ nozdormism_adherent:0 "Nozdormist"
+ nozdormism_adherent_plural:0 "Nozdormists"
+ nozdormism_desc:0 "The followers of the Bronze Dragonflight are the guardians of time and the keepers of history. They believe that the flow of time is sacred, and that the wise Aspect of Time has entrusted the bronze dragons with the duty of safeguarding it. They seek to understand the complex threads of destiny and to protect the past, present, and future from any attempt to alter the timeline. The Hourglass of Time is a powerful artifact that the bronze dragons use to subtly manipulate time. For the followers of the Bronze Dragonflight, time is not just a linear concept, but a fluid and mutable force that must be revered and preserved."
```

Comment:
```suggestion
 nozdormism_desc:0 "Of all the Dragons, the Bronze Dragonflight is most poorly understood, and rightly so. Their domain, which they view as a sacred mission entrusted to them by the Aspect of Time, is the safeguarding of chronological and causal order and the integrity of the underlying mechanics of the universe. This puts the Bronze Dragons far beyond the understanding of even the most intelligent mortals, despite being broadly seen as the friendliest and most convivial with lesser beings on the rare occasion they are able to meet one. Thus, mortal followers of the Bronze Dragonflight take their inscrutable utterances and mysterious doings as proof of their ultimate claim on Divinity through the ancient Titans. Because of their poor understanding of their living deities, mortal followers of the Bronze Dragonflight are strong believers in fate and predestination, seeing themselves as being at the mercy of an inscrutable cosmos."
```